<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dht11</class>
 <widget class="QMainWindow" name="Dht11">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>228获取温湿度</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="hlabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>温度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="humidityLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="temperatureLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>获取</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
